Dockerfile

```
@@ -2,162 +1,100 @@
+###############################################################################
+# 2. Build Stage
+###############################################################################
+FROM deps AS build
+
+# Accept build arg for versioning
+ARG APP_VERSION=0.1.0-docker
+ENV APP_VERSION=${APP_VERSION}
+ENV MIX_ENV=prod
+
+# Install Node.js for asset compilation
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)